cmd/tailscale/cli/up.go

```diff
@@ -439,6 +439,9 @@ func runUp(ctx context.Context, cmd string, args []string, upArgs upArgsT) (retE
 		if err := localClient.CheckIPForwarding(context.Background()); err != nil {
 			warnf("%v", err)
 		}
+		if err = localClient.CheckUDPGROForwarding(context.Background()); err != nil {
```

Comment:
there's already a `ctx` in scope here from L389. Use that here (and on L439?) while you're here?

Reply:
Updated in https://github.com/tailscale/tailscale/pull/10071/commits/286d81b20cbc486f91bbe5227ea6958c3f23313f